src/blob-api.js

```diff
@@ -64,8 +64,8 @@ export class BlobApi {
         variant: 'original',
         type: blobType,
       },
-      metadata,
-      contentHash
+      metadata
+      // contentHash
```

Comment:
This fixes the e2e media-server test, but it breaks the blob-api test because the hash is incorrect.